sonar-project.properties

```diff
@@ -1,10 +0,0 @@
-sonar.projectKey=open-cluster-management_thanos
```

Comment:
Is this from some previously used CI?

Reply:
Yes, we no longer run sonarCloud on maintenance forks.